### PR TITLE
Officer cloaks become satchels

### PR DESCRIPTION
**Pull request by grinkgo**
## About The Pull Request

Officer cloak (normal and red) and Captain's cloak (normal and red) changed from being equipped to the suit slot to the back slot. They also have a satchel's worth of storage.

## Why It's Good For The Game

Other cloaks like the sniper smock and scout cloak are worn on the back and have storage slots. Making the command cloaks into backpacks gives them utility in addition to pure aesthetic. Additionally, it allows for officers to be able to wear their armor in their suit slot as well as their cloak, if they wish.

## Changelog
:cl:
balance: The officer and captain's cloak are now worn on the back and have limited storage, so you can have both aesthetic AND utility!
/:cl:

### CODE REVIEW

**Comment by psykzz** [comment]
balance patch yo, where them balance tags.

**Comment by DominikPanic** [comment]
Yeah this should be a balance change, you are basically making them into regular backpacks with this change, and they will have a huge storage space.

**Comment by Rohesie** [comment]
Not bad at all.

File: code/game/objects/machinery/vending/new_marine_vendors.dm [hunk 1]
```diff
@@ -730,6 +728,8 @@
 							list("TGMC technician backpack", 0, /obj/item/storage/backpack/marine/tech , MARINE_CAN_BUY_BACKPACK, "black"),
 							list("TGMC technician welderpack", 0, /obj/item/storage/backpack/marine/engineerpack , MARINE_CAN_BUY_BACKPACK, "black"),
 							list("Lightweight combat pack", 0, /obj/item/storage/backpack/lightpack, MARINE_CAN_BUY_BACKPACK, "black"),
+							list("Officer cloak", 0, /obj/item/storage/backpack/marine/satchel/officer_cloak, MARINE_CAN_BUY_BACKPACK, "black"),
+							list("Officer cloak, red", 0, /obj/item/storage/backpack/marine/satchel/officer_cloak_red, MARINE_CAN_BUY_ARMOR, "black"),
```
Comment (psykzz) [inline code/game/objects/machinery/vending/new_marine_vendors.dm]:
```suggestion
							list("Officer cloak, red", 0, /obj/item/storage/backpack/marine/satchel/officer_cloak_red, MARINE_CAN_BUY_BACKPACK, "black"),
```

**Pull request merged**